tensorflow/compiler/xla/service/gpu/gpu_conv_algorithm_picker.cc

```diff
@@ -138,15 +297,350 @@ StatusOr<AutotuneResult> GpuConvAlgorithmPicker::PickBestAlgorithm(
     allocator = &*se_allocator;
   }
 
-  StatusOr<AutotuneResult> result_or =
-      PickBestAlgorithmNoCache(*instr, allocator, &stream);
+  StatusOr<AutotuneResult> result_or(InternalError("Unknown platform."));
+  if (stream_exec_->platform_kind() == se::PlatformKind::kROCm) {
```

Comment:
@jerryyin might need some comments here saying we check StreamExecutor implementation here. And on ROCm path for now we must ensure no redzone allocator would be used.

Reply:
Will do